sectorbuilder.go

```diff
@@ -445,30 +444,22 @@ func sizeFromConfig(cfg Config) (abi.SectorSize, error) {
 
 func sizeFromProofType(p abi.RegisteredProof) (abi.SectorSize, error) {
 	switch p {
-	case abi.RegisteredProof_WinStackedDRG32GiBSeal:
-		return 1 << 35, nil
-	case abi.RegisteredProof_WinStackedDRG32GiBPoSt:
-		return 1 << 35, nil
 	case abi.RegisteredProof_StackedDRG32GiBSeal:
-		return 1 << 35, nil
+		return 32 << 20, nil
```

Comment:
It takes a fair bit of scrutiny and understanding to understand while we're shifting a power of 2 by a multiple of 10. Defining constants `MB := 1<<20` and `KB := 1<<10` and then returning `32*MB`, `2*KB`, `8*MB` etc. would clarify considerably.